### DISCUSSION TITLE
php on ios

### DISCUSSION THREAD

**Discussion by odarcan** [crazywhalecc/static-php-cli#316] [Ideas]
hi @crazywhalecc do you know how to compile php for ios and embed it into an app?

**Comment by crazywhalecc** [comment]
I have never developed an iOS App, so I don’t know much about it, to use it on iOS, probably php should be statically linked to OC or Swift

**Comment by simonhamp** [comment]
@odarcan the ARM64 build of PHP works on iOS devices as they share the same architecture with modern Apple computers.

I also have never developed an iOS app, but I know that it can be done. Marcel Pociot put a proof of concept together a while back and shared it on Twitter (I can't find the tweet now).

But you should consider that it's very unlikely (like 99.99% certain) that Apple will not allow an interpreter embedded in your application to be published through its app stores.
